### ISSUE TITLE
a problem with smart TVs

### ISSUE THREAD

**Issue by Mohammadnaghi** [quran/quran_android#945]
hello all friends
i use this good app in my smart tvbox.
there is a problem with android boxs & tvs, the heder & footer don't hide and always show up's icons and qari's name in buttom. so end of pages didn't seen! 
in my smart phone the heder and footer after 2 secend automaticaly hide!
any body can help?

**Comment by Mohammadnaghi** [comment]
thanks dear ahmedre

![photo_ - - _ - -](https://user-images.githubusercontent.com/38376730/38791760-c2f215b0-415e-11e8-9189-428b92158488.jpg)

my tv is not touch screen and i use mouse to play app.i think if you set app automatic hide heder and footer without checking anything, it will work inshaAllah.
i try to set it myself, but i cant import project. the gradle can't download some filse.

![2](https://user-images.githubusercontent.com/38376730/38792339-c1d0b404-4161-11e8-9e34-5412e34fd244.jpg)

there is error:
No cached version of com.android.tools.build:gradle:3.2.0-alpha09 available for offline mode.
and
Could not GET 'https://maven.fabric.io/public/com/android/tools/build/gradle/3.2.0-alpha09/gradle-3.2.0-alpha09.pom'. Received status code 403 from server: Forbidden

**Comment by ozbek** [comment]
As suggested in the build logs, disable offline mode and try again.

**Comment by Mohammadnaghi** [comment]
thanks ozbek
i tried it a lot and the same error was:
Could not GET 'https://maven.fabric.io/public/com/android/tools/build/gradle/3.2.0-alpha09/gradle-3.2.0-alpha09.pom'. Received status code 403 from server: Forbidden

**Comment by ozbek** [comment]
Can you try with the latest changes, please? We updated the gradle version.

**Comment by Korogba** [comment]
I had a look at this. I was able to replicate using a TV emulator but hiding the status bar after a few seconds works well on a device but not on the TV emulator.

Reading the [Android TV Guide](https://developer.android.com/training/tv/start/layouts#anti-patterns), it is strongly discouraged to re-use phones/tablets layouts for TVs. Secondly, even if the app is installed, most users will not be able to use it because there is no [TV launcher activity](https://developer.android.com/training/tv/start/start#tv-activity). 

I suggest creating a separate module in the app and designing specifically for TVs. Are there designs available already? This is something I am willing to pick up, if I have proper designs.

**Comment by ahmedre** [comment]
makes sense - we currently don't have any designs or anything. i'd be curious to learn more about the use case for this.

**Comment by rababR1** [comment]
Assalamu Alaykum, its been 4 years....
How's the progress going?
